apis/quay/v1/quayregistry_types.go

```diff
@@ -30,14 +30,23 @@ import (
 
 type QuayVersion string
 
-const (
-	QuayVersionCurrent  QuayVersion = "vader"
-	QuayVersionPrevious QuayVersion = ""
-)
+const QuayVersionCurrent QuayVersion = "3.4.3"
+const QuayVersionPrevious QuayVersion = "vader"
+
+var upgradeable = []QuayVersion{
+	QuayVersionCurrent,
+	"vader",
+	"",
+}
```

Comment:
i don't like this... this is the main branch and both redhat-3.4 and redhat-3.5 will need custom values... how will that be handled?

Reply:
Ideally we use environment variables set during the Operator's container build. Is that something easy to accomplish?